plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/utils/UIUtils.java

```diff
@@ -505,6 +509,19 @@ public boolean select(Viewer viewer, Object parentElement, Object element) {
         return dialog;
     }
 
-
+    /**
+     * Use this method instead of HandlerUtil.getCurrentSelection(event)
+     * when action is called on Properties View.
+     * 
+     * @param event
+     * @return
+     */
+    public static ISelection getCurrentSelection(ExecutionEvent event) {
```

Comment:
What is the rationale for this method instead of HandlerUtil.getCurrentSelection ?